src/Binder/CustomBinder.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Tsuka\DB\Binder;
+
+use Katana\Sdk\Action;
+use Tsuka\DB\Entity;
+
+/**
+ * Sets either a multiple or a single relation of an Entity with a custom id field.
+ *
+ * The Entity must have an id property to set the relationship.
+ *
+ * The custom id or ids are looked for in an Entity field with a custom relation name.
+ *
+ * This Binder relies on a service named as the relation with a custom action
+ * that accepts such custom parameter with a comma separated list.
+ *
+ * @package Tsuka\DB\Binder
+ */
+class CustomBinder implements BinderInterface
+{
+
+    public $actionName = '';
```

Comment:
Please, can you add simple docblocks for properties? Just with `@var string` or `@var bool`